src/ProxyClient/HttpDispatcher.php

```diff
@@ -187,52 +192,65 @@ protected function getServers()
      */
     private function fanOut(RequestInterface $request)
     {
-        $requests = [];
-
-        $uri = $request->getUri();
-
-        // If a base URI is configured, try to make partial invalidation
-        // requests complete.
-        if ($this->baseUri) {
-            if ($uri->getHost()) {
-                // Absolute URI: does it already have a scheme?
-                if (!$uri->getScheme() && '' !== $this->baseUri->getScheme()) {
-                    $uri = $uri->withScheme($this->baseUri->getScheme());
-                }
-            } else {
-                // Relative URI
-                if ('' !== $this->baseUri->getHost()) {
-                    $uri = $uri->withHost($this->baseUri->getHost());
-                }
-
-                if ($this->baseUri->getPort()) {
-                    $uri = $uri->withPort($this->baseUri->getPort());
+        /** @var RequestInterface[] $requests */
```

Comment:
we should not need this comment, we only put request interface into this variable and code checkers should see that i think.

Reply:
Not phpstorm.